apps/testclient/management/commands/create_test_users_and_applications_batch.py

```diff
@@ -94,33 +116,36 @@ def create_dev_users_apps_and_bene_crosswalks(group, bene_count, dev_count, app_
                         "email": fn + '.' + ln + "@xyz.net",
                     }
                     slsx_client = OAuth2ConfigSLSx(args)
-                    u = create_beneficiary_record(slsx_client, fhir_id)
-                    date_picked = datetime.utcnow() - timedelta(days=randrange(700))
-                    u.date_joined = date_picked.replace(tzinfo=pytz.utc)
-                    u.save()
-                    c = Crosswalk.objects.get(user=u)
-                    c.created = u.date_joined
-                    c.save()
-                    bene_pk_list.append(u.pk)
-                    synthetic_bene_cnt += 1
-                    print(".", end="", flush=True)
-                    time.sleep(.05)
-                    if count > bene_count:
-                        break
+                    try:
+                        u = create_beneficiary_record(slsx_client, fhir_id)
+                        date_picked = datetime.utcnow() - timedelta(days=randrange(700))
+                        u.date_joined = date_picked.replace(tzinfo=pytz.utc)
+                        u.save()
+                        c = Crosswalk.objects.get(user=u)
```

Comment:
what does Crosswalk.objects.get(user=u) do? Is that checking if the user already exists?

Reply:
It's fetched so that the created date can be updated to be the same as the user data_joined. in the next line.

Reply:
@sharonfruit  Okay that makes more sense but could you explain what is meant by "Crosswalk?" I've heard it in different contexts throughout my rotations

Reply:
Crosswalk name is a good mataphor.  So like one at a street it's basically crossing the street between the mbi and the fhir id.  So it just maps those 2 together so that we know which fhir record goes with which bene.

Reply:
@ajshred ah! Thank you for clarifying!